docs/doc_templates/lorenz-example.md

```diff
@@ -0,0 +1,216 @@
+---
+# Page settings
+layout: default
+keywords: lorenz example
+comments: false
+permalink: /docs/examples/lorenz/
+
+# Hero section
+title: Lorenz Example
+description: A more complex example with a chaotic system
+
+# Micro navigation
+micro_nav: true
+
+# Page navigation
+page_nav:
+    prev:
+        content: Previous page
+        url: '/docs/examples/thermometer/'
+---
+
+## Overview
+
+This example aims to demonstrate two things:
+
+* using Keanu with [Kotlin](https://kotlinlang.org/)
+* how to solve a time-stepped problem
+
+## The problem to model
+
+Imagine you are watching the point below moving through space:
+
+![Trajectory through phase space](https://upload.wikimedia.org/wikipedia/commons/1/13/A_Trajectory_Through_Phase_Space_in_a_Lorenz_Attractor.gif)
+
+You'd begin to notice it has an interesting repeated pattern of movement. If you had a really
+keen eye, you'd notice that its movement can be described by the
+[Lorenz equations](https://en.wikipedia.org/wiki/Lorenz_system).
+
+The Lorenz equations are particuarly hard to model as they describe a chaotic system. Two initial
+starting states, no matter how close, will diverge almost immediately. 
+
+In this example, you'll make observations on the point above and build a probabilistic model
+using Keanu that can accurately describe its chaotic motion. To make things harder, you are going to
+only observe the X coordinate, and to observe a noisy value of the X coordinate.
+
+### How to model the problem
+
+To start with, you'll need to build a simple program that can step through time and calculate
+coordinates using the Lorenz equations. This will serve as the "Real World" system that you
```

Comment:
This will serve as the “Real World” system from which you can take observations of the X coordinate, and against which you can compare the calculated coordinates of the probabilistic model.